skbeam/core/accumulators/binned_statistic.py

```diff
@@ -70,8 +70,13 @@ def __init__(self, sample, statistic='mean',
             A sequence of lower and upper bin edges to be used if the
             edges are not given explicitely in `bins`. Defaults to the
             minimum and maximum values along each dimension.
-
+        mask : array_like
```

Comment:
Hi Julien,

As Tom also mentioned, I inherited this code/documentation from scipy's binned_statistic.  To me the order appears correct since in the code it says:

        # Sample is an ND-array.                                                                                              
        N, D = sample.shape